Source/WebKit/UIProcess/WebAuthentication/fido/CtapAuthenticator.cpp

```diff
@@ -117,6 +124,7 @@ void CtapAuthenticator::continueMakeCredentialAfterResponseReceived(Vector<uint8
     auto response = readCTAPMakeCredentialResponse(data, AuthenticatorAttachment::CrossPlatform, transports(), std::get<PublicKeyCredentialCreationOptions>(requestData().options).attestation);
     if (!response) {
         auto error = getResponseCode(data);
+        WTFLogAlways("got error %d", (int)error);
```

Comment:
You probably want this to be a DEBUG log message, rather than spamming all the time.

Reply:
It's now a RELEASE_LOG_DEBUG. Thank you for the review!